subsys/bluetooth/controller/Kconfig.dtm

```diff
@@ -0,0 +1,49 @@
+# Zephyr Bluetooth Controller configuration options
+
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config BT_CTLR_DTM_HCI_SUPPORT
+	bool
+
+config BT_CTLR_DTM
+	bool
+	help
+	  Enable support for Direct Test Mode in the Controller.
+
+menuconfig BT_CTLR_DTM_HCI
+	bool "Direct Test Mode over HCI"
+	depends on BT_CTLR_DTM_HCI_SUPPORT
+	select BT_CTLR_DTM
+	help
+	  Enable support for Direct Test Mode over the HCI transport.
+
+if BT_CTLR_DTM_HCI
+
+config BT_CTLR_DTM_HCI_RX_V3
```

Comment:
```suggestion
if BT_CTLR_DTM_HCI

if BT_CTLR_DF_SUPPORT

config BT_CTLR_DTM_HCI_RX_V3
```

Reply:
I am not sure about adding `BT_CTLR_DF_SUPPORT` dependency here. Those command can be still used in case if controller doesn't support DF. The spec has `Missing parameters` paragraph for those command that describes how to pass parameters when we for example don't want to use CTE. The transmitter command in version 4 is the only command which can set Tx power level, in my opinion it can be used even if Controller doesn't support DF and CTE length is set to 0, if the CTE length will be set to value other that 0 then it returns HCI ERROR CMD DISALLOWED. What do you thing about it ?

Reply:
fair enough.